src/sentry/snuba/discover.py

```diff
@@ -1325,6 +1325,69 @@ def get_facets(
     return results
 
 
+def spans_histogram_query(
+    span,
+    user_query,
+    params,
+    num_buckets,
+    precision=0,
+    min_value=None,
+    max_value=None,
+    data_filter=None,
+    referrer=None,
+    group_by=None,
+    order_by=None,
+    limit_by=None,
+    extra_snql_condition=None,
+    normalize_results=True,
+):
```

Comment:
You might want a doc block here describing these params (like histogram on https://github.com/getsentry/sentry/pull/32529/files#diff-f9a8d9bbcb1bf33b928235196e41ca3666cc8c508bbb46d3238aae26d3e13a6cR1411)

Reply:
done 👍